packages/babel-preset-env/src/index.ts

```diff
@@ -170,6 +170,9 @@ export const getModulesPluginNames = ({
     modulesPluginNames.push("syntax-top-level-await");
   }
 
+  // Enable import meta for @babel/core < 7.10
+  modulesPluginNames.push("syntax-import-meta");
```

Comment:
Let's defer the changes to Babel 8.1 so we don't have to maintain more Babel 7 vs Babel 8 cases. The change can be safely landed within 8.0 to 8.1, at that time we can update test fixtures in batch without duplicating Babel 7 cases from Babel 8.